content/blog/en/version-6-dojo.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Announcing Dojo 6
+date: 2019-07-30T08:00:00.000Z
+author: Anthony Gubler
+---
+
+Since the first major Dojo release last year, we have been working to refine the features and patterns to make Dojo an even more efficient framework for building applications with TypeScript and modern web APIs.
+
+In a world of semantic versioning where even minor breaking changes require a new version number, it is challenging to know when a new version is substantial. Today, we’re excited to announce version 6, our most ambitious set of improvements since the Dojo 2.0 release.
+
+Version 6 of Dojo brings many new features and changes to substantially improve the development experience when creating applications with Dojo by reducing boilerplate, increasing flexibility, and further improving performance.
+
+![The image for the blog](assets/blog/version-6-dojo/featured.png)
+<!-- more -->
+
+## Function Based Widgets and Middleware
```

Comment:
Not sure if we want to align on Sentence-style capitalization for headers, as per the ref guides

Reply:
I wasn't sure if blogs were different as I had a look at a couple of different release blogs and they seemed to have capitalised headings.

Reply:
Also all the previous blogs followed the same pattern

Reply:
Yeah i suppose consistency with other blog entries trumps newer parts of the site.

Reply:
We could always update the other blog entries.